### ISSUE TITLE
Missing images fluent-bit:1.8.15-windows-1809 and mirrored-pause:3.6-windows-1809 on Docker Hub

### ISSUE THREAD

**Issue by jwhb** [rancher/rancher#37279]
**Edit:** This issue is about addressing follow up comment: https://github.com/rancher/rancher/issues/37279#issuecomment-1172219838

**Rancher Server Setup**
- Rancher version: 2.6.4
- Installation option (Docker install/Helm Chart):
   - RKE2 v1.23.5+rke2r1 Air-Gap
- Proxy/Cert Details: n/a

**Information about the Cluster**
- Kubernetes version: 1.23.5
- Cluster Type (Local/Downstream):
   - Local, rancher-server chart not installed yet
 * Infrastructure Provider = Azure VMs == "bare-metal"

**User Information**
- n/a, Rancher Server not installed yet

   
**Describe the bug**

Mirroring of images in rancher-images-windows.txt fails for Rancher 2.6.4, images `rancher/fluent-bit:1.8.15-windows-1809` and `rancher/mirrored-pause:3.6-windows-1809` not available on Docker Hub. They are required for Air-Gap setup of Rancher 2.6.4.

**To Reproduce**
`docker pull docker.io/rancher/fluent-bit:1.8.15-windows-1809` or `docker manifest inspect docker.io/rancher/fluent-bit:1.8.15-windows-1809`. Same for `rancher/mirrored-pause:3.6-windows-1809`

To check availability of all Rancher Windows images for 2.6.4:
```bash
$ curl -s -L https://github.com/rancher/rancher/releases/download/v2.6.4/rancher-windows-images.txt | while read IMAGE; do
    docker manifest inspect $IMAGE-windows-1809 > /dev/null 2>&1 || echo "not found in registry: $IMAGE-windows-1809"
done
not found in registry: rancher/fluent-bit:1.8.15-windows-1809
not found in registry: rancher/mirrored-pause:3.6-windows-1809
```

**Result**

Docker fails to pull fluent-bit and pause images.
```bash
$ docker pull docker.io/rancher/fluent-bit:1.8.15-windows-1809 
Error response from daemon: manifest for rancher/fluent-bit:1.8.15-windows-1809 not found: manifest unknown: manifest unknown
```

**Expected Result**
Docker (or other tools like podman, crane) successfully pulls fluent-bit and pause images for Windows.

**Screenshots**

**Additional context**
Following https://rancher.com/docs/rancher/v2.6/en/installation/other-installation-methods/air-gap/populate-private-registry/ to gather Windows images to push to a registry in air-gap environment.

For lack of an internet-connected Windows OS I'm not using the `rancher-save-images.ps1`. I use crane and podman instead. Either way the images are not available on docker.io.

SURE-4711

**Comment by asemen2608** [comment]
Mirroring of images in rancher-images-windows.txt fails for Rancher 2.6.5, 
Images rancher/pause:3.6-windows-1809-amd64 and rancher/pause:3.6-windows-ltsc2022-amd64 are not available on Docker Hub. They are required for Air-Gap setup of Rancher 2.6.5.

```
# docker pull rancher/pause:3.6-windows-1809-amd64
3.6-windows-1809-amd64: Pulling from rancher/pause
bc8517709e9c: Pulling fs layer 
9b4754cb2394: Download complete 
6851284b6a3b: Downloading [>                                                  ]   14.3kB/1.402MB
f3395efd3035: Waiting 
image operating system "windows" cannot be used on this platform

# docker pull rancher/pause:3.6-windows-ltsc2022-amd64
3.6-windows-ltsc2022-amd64: Pulling from rancher/pause
ee7a9c9193e3: Downloading [>                                                  ]    539kB/118MB
784aba008ad8: Download complete 
02e02a05e018: Downloading [>                                                  ]  14.32kB/1.402MB
0d37d6f9240b: Waiting 
```

**Comment by asemen2608** [comment]
Not yet fixed in Rancher v2.6.6 rancher-windows-images.txt

curl -s -L https://github.com/rancher/rancher/releases/download/v2.6.6/rancher-windows-images.txt | while read IMAGE; do     docker manifest inspect $IMAGE-windows-1809 > /dev/null 2>&1 || echo "not found in registry: $IMAGE-windows-1809"; done
not found in registry: rancher/fluent-bit:1.8.15-windows-1809
not found in registry: rancher/mirrored-pause:3.6-windows-1809

docker pull docker.io/rancher/pause:3.6-windows-1809-amd64
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
asemen@data:~/case/00347632/v2.6.6> docker pull docker.io/rancher/pause:3.6-windows-1809-amd64
3.6-windows-1809-amd64: Pulling from rancher/pause
bc8517709e9c: Downloading [>                                                  ]  529.3kB/102.7MB
9b4754cb2394: Download complete 
6851284b6a3b: Download complete 
f3395efd3035: Waiting 
image operating system "windows" cannot be used on this platform 

docker pull rancher/pause:3.6-windows-ltsc2022-amd64
3.6-windows-ltsc2022-amd64: Pulling from rancher/pause
ee7a9c9193e3: Pulling fs layer 
784aba008ad8: Pulling fs layer 
02e02a05e018: Pulling fs layer 
0d37d6f9240b: Waiting 
image operating system "windows" cannot be used on this platform

**Comment by deniseschannon** [comment]
Windows 1809 dropped support as of last year, please use a valid Windows server version. 

Based on you running 2.6.4, here is the support matrix for your version of Rancher:
https://www.suse.com/assets/EN-Rancherv2.6.4-150422-0151-56.pdf
